include/zrepl_mgmt.h

```diff
@@ -92,7 +92,9 @@ typedef struct zvol_info_s {
 	/* Logical Unit related fields */
 	zvol_info_state_t	state;
 	char 		name[MAXPATHLEN];
-	zvol_state_t	*zv;
+	zvol_state_t	*original_zv;
```

Comment:
lets change the name to volume_zv or vol_zv or prime_zv or master_zv or main_zv or first_zv.. or just zv also should be fine

Reply:
Done